app/views/casework/crime_applications/sections/_income.html.erb

```diff
@@ -1,14 +1,16 @@
 <% if income_details.present? %>
   <%= govuk_summary_card(title: label_text(:income)) do %>
     <dl class="govuk-summary-list">
```

Comment:
@RoseSAK We need to have a check on <dl> else it will show blank income section. 

<img width="924" alt="Screenshot 2024-07-17 at 14 37 10" src="https://github.com/user-attachments/assets/7252f33a-7f15-4d8d-9711-e96bba6f7339">